### ISSUE TITLE
Work great but need fix with other tweaks

### ISSUE THREAD

**Issue by rbasrawi** [ioscreatix/Intelix#20]
Pls make it compatible with the following tweaks:

MoreLinesNotifications: 1.0.0-4
Palette: 0.3-1+debug

**Comment by andrewwiik** [comment]
Tweak compatability is not a goal at the currently time, currently I am mostly interested in crashes

**Comment by sirbc** [comment]
Yes. More lines and palette are a must. So nice. Just would like them grouped.

**Comment by sirbc** [comment]
Got colorbanners2. You the man! Thanks

BRANDON CAREY
Mobile: 704–663–9121
